content/post/2022-09-01-r-dependency/index.Rmd

```diff
@@ -0,0 +1,223 @@
+---
+slug: r-dependency
+title: "Checking the inputs of your R functions" 
+authors: 
+- Hugo Gruson
+- Maëlle Salmon
+date: "2022-09-01" 
+tags: 
+- package development 
+- r-package
+output: hugodown::hugo_document
+---
+
+```{r setup, include=FALSE}
+
+knitr::opts_chunk$set(fig.path = "", comment = "")
+# knitr hook to make images output use Hugo options
+knitr::knit_hooks$set(
+  plot = function(x, options) {
+    hugoopts <- options$hugoopts
+    paste0(
+      "{{<figure src=",
+      '"', x, '" ',
+      if (!is.null(hugoopts)) {
+        glue::glue_collapse(
+          glue::glue('{names(hugoopts)}="{hugoopts}"'),
+          sep = " "
+        )
+      },
+      ">}}\n"
+    )
+  }
+)
+options(crayon.enabled = FALSE)
+
+```
+
+There have been much talk and many blog posts about R package dependencies.
+Yet, one special dependency is more rarely mentioned, even though all packages include it: the dependency on R itself.
+The same way you can specify a dependency on a package, and optionally on a specific version, you can add a dependency to a minimum R version in the `DESCRIPTION` file of your package.
+In this blog post, we will explain:
+
+-   How to declare a dependency to a minimum R version and why you may need to do so?
```

Comment:
I feel that at this point the questions could be compacted a bit as the structure is fairly clear. So I'd suggest shortening this part of the intro.